rlkit/core/rl_algorithm.py

```diff
@@ -175,45 +174,23 @@ def train(self):
                 for idx in self.train_tasks:
                     self.task_idx = idx
                     self.env.reset_task(idx)
-                    self.collect_data_sampling_from_prior(num_samples=self.num_initial_steps,
-                                                          resample_z_every_n=self.max_path_length,
-                                                          eval_task=False)
-
+                    self.collect_data(self.policy, self.num_initial_steps, 1, np.inf)
```

Comment:
I kind of dislike using `np.inf` to get samples from the prior only, but not sure what the cleaner way should look like?

Reply:
I t hink np inf is fine.

Reply:
let's roll with it for now